src/visualize/visualize_filters_activations.py

```diff
@@ -1,6 +1,10 @@
 # Imports
 
 # > Standard Library
+import matplotlib.pyplot as plt
+import tensorflow as tf
+import numpy as np
+from vis_utils import prep_image_for_model, init_pre_trained_model
```

Comment:
tf, plt and numpy are not in standard library.  Maybe you can fix the other imports here as well? =^)